test_try_lock_informatively fails on github actions for datalad/git-annex

freshly added test fails... most likely depends are missing, so no PIDs but test assumes that it has it

https://github.com/datalad/git-annex/runs/3367619239?check_suite_focus=true

```
FAIL: datalad.support.tests.test_locking.test_try_lock_informatively
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/datalad/tests/utils.py", line 764, in _wrap_with_tempfile
    return t(*(arg + (filename,)), **kw)
  File "/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/datalad/support/tests/test_locking.py", line 153, in test_try_lock_informatively
    assert_in('Check following process: PID=', res['stderr'])
AssertionError: 'Check following process: PID=' not found in '[INFO] Failed to acquire lock at /tmp/datalad_temp_test_try_lock_informativelytzjs8vu0.lck in 0.05 seconds. Another process is using it (failed to determine one)? Will try again and wait for up to 0.15 seconds. \n[INFO] Failed to acquire lock at /tmp/datalad_temp_test_try_lock_informativelytzjs8vu0.lck in 0.15 seconds. Another process is using it (failed to determine one)? Will proceed without locking. \n'
-------------------- >> begin captured logging << --------------------
fasteners.process_lock: Level 5: Acquired file lock `b'/tmp/datalad_temp_test_try_lock_informativelytzjs8vu0.lck'` after waiting 0.000s [1 attempts were required]
fasteners.process_lock: Level 5: Unlocked and closed file lock open on `b'/tmp/datalad_temp_test_try_lock_informativelytzjs8vu0.lck'`
```

--- a/datalad/support/tests/test_locking.py
+++ b/datalad/support/tests/test_locking.py
@@ -149,9 +149,12 @@ with try_lock_informatively(lock, timeouts=[0.05, 0.15], proceed_unlocked={{proc
         assert_in('proceed without locking', res['stderr'])
         assert_greater(time() - t0, 0.19999)  # should wait for at least 0.2
         try:
+            import psutil
             # PID does not correspond
             assert_in('Check following process: PID=', res['stderr'])
             assert_in(f'CWD={os.getcwd()} CMDLINE=', res['stderr'])
+        except ImportError:
+            pass  # psutil was not installed, cannot get list of files
         except AssertionError:
             # we must have had the other one then
             assert_in('failed to determine one', res['stderr'])

